app/assets/stylesheets/style.scss

```diff
@@ -506,42 +506,43 @@ ul.quiet_list {
     background: #fef7e0;
   }
 }
-form.edit_lightweight_activity {
+form.edit_lightweight_activity,
+form.new_lightweight_activity {
```

Comment:
This is the only new content in the file - the rest is space deletion.  The new form was unstyled.